dynamic/dynamic_message_test.go

```diff
@@ -1952,6 +1953,48 @@ func TestMergeFrom(t *testing.T) {
 	// TODO
 }
 
+func TestBuiltMessage(t *testing.T) {
+	md, err := builder.NewMessage("Person").
+		AddField(builder.NewField("name", builder.FieldTypeString())).
+		Build()
+
+	if err != nil {
```

Comment:
If you look at most of the tests, you'll see they use `testutil.Ok` for this sort of check:

```go
testutil.Ok(t, err, "failed to load message descriptor")
```